### PR TITLE
[mac] fixes date control inconsistent

### PR DESCRIPTION
**Pull request by paymicro**
Context [AB#1137572](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1137572)
image

### CODE REVIEW

File: Xamarin.PropertyEditing.Mac/Controls/DateTimeEditorControl.cs [hunk 1]
```diff
@@ -16,7 +16,8 @@ public DateTimeEditorControl (IHostResourceProvider hostResources)
 				DatePickerElements = NSDatePickerElementFlags.HourMinuteSecond | NSDatePickerElementFlags.YearMonthDateDay,
 				DatePickerStyle = NSDatePickerStyle.TextFieldAndStepper,
 				Font = NSFont.SystemFontOfSize (NSFont.SystemFontSizeForControlSize (NSControlSize.Small)),
-				TranslatesAutoresizingMaskIntoConstraints = false
+				TranslatesAutoresizingMaskIntoConstraints = false,
+				TimeZone = Foundation.NSTimeZone.FromAbbreviation ("GMT")
```
Comment (CartBlanche) [inline Xamarin.PropertyEditing.Mac/Controls/DateTimeEditorControl.cs:20]: Sorry about the late reply. Just wondering why we want to hard code this to "GMT"?  
Could you also give an example of the inconsistency you are seeing, so I have something to compare to?

Comment (netonjm) [reply Xamarin.PropertyEditing.Mac/Controls/DateTimeEditorControl.cs:20]: This should be UTC not GMT.


image

Comment (netonjm) [reply Xamarin.PropertyEditing.Mac/Controls/DateTimeEditorControl.cs:20]: This should be UTC, not GMT. Add the abbreviation into a constant.

9qYGQXRAjZ

Comment (netonjm) [reply Xamarin.PropertyEditing.Mac/Controls/DateTimeEditorControl.cs:20]: As the recent conversation this doesn't fix the real problem in Xam.Forms and it needs to be addressed in different way, adding some customisation to allow, in addition to full date time, only time or date.

**Pull request closed without merging**